tests/groupby_test.py

```diff
@@ -440,6 +441,41 @@ def test_zero_length_groupby(self):
         g = ak.GroupBy(ak.zeros(0, dtype=ak.int64))
         str(g.segments)  # passing condition, if this was deleted it will cause the test to fail
 
+    def test_first_aggregation(self):
+        keys = ak.array([0, 1, 0, 1, 0, 1])
+        vals = ak.array([9, 8, 7, 6, 5, 4])
+        ans = [9, 8]
+        g = ak.GroupBy(keys)
+        _, res = g.first(vals)
+        self.assertListEqual(ans, res.to_ndarray().tolist())
```

Comment:
you should be able to use the new `to_list()` functionality that runs `.to_ndarray().tolist()` under the hood. @pierce314159 added this recently to make tests easier to read.